### ISSUE TITLE
ODT->PDF Conversion: Picture Description shows up in PDF

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dbarra/xdocreport#191]
```
>>>What steps will reproduce the problem?
1. Convert the attached "image_description.odt" file to PDF using the 
PdfConverter XDocReport class.

>>> What is the expected output? What do you see instead?
The expected output should be just the image. In the pdf the image is shown 
along with the text of the picture description.

>>> What version of the product are you using? On what operating system?
1.0.0-SNAPSHOT, latest code from the git repo (as of this morning.)
Windows 7 64-bit

>>> Please provide any additional information below.
I know that the easy workaround for this would just be to remove the picture 
description, but I think the normal behavior here should probably *not* display 
the description.
```

Original issue reported on code.google.com by `PhilipDN...@gmail.com` on 9 Nov 2012 at 7:22

Attachments:
- [image_description.odt](https://storage.googleapis.com/google-code-attachments/xdocreport/issue-191/comment-0/image_description.odt)
- [image_description.pdf](https://storage.googleapis.com/google-code-attachments/xdocreport/issue-191/comment-0/image_description.pdf)

**Comment by GoogleCodeExporter** [comment]
```
@Leszek : I have add a JUnit 
http://code.google.com/p/xdocreport/source/browse/thirdparties-extension/org.odf
toolkit.odfdom.converter.pdf/src/test/resources/org/odftoolkit/odfdom/converter/
core/Issue191.odt with the odt attached of this issue.
```

Original comment by `angelo.z...@gmail.com` on 10 Nov 2012 at 2:10
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
fixed in version 1.0.0
```

Original comment by `abe...@gmail.com` on 13 Dec 2012 at 8:05
- Changed state: **Fixed**
